tests/rbd_mirror/test_mirror_promote.py

```diff
@@ -0,0 +1,54 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """To promote secondary cluster as primary after primary cluster fails,
+    which could be used for Regional DR scenario.
+
+    Args:
+        **kw: test data
+        Example::
+            config:
+                imagesize: 2G
+    Returns:
+        0 - if test case pass
+        1 - if test case fails
+    """
+    try:
+        log.info("Running test CEPH-9476, Promoting secondary cluster as primary")
+        config = kw.get("config")
+        poolname = config.get("poolname")
+
+        mirror2 = rbdmirror.RbdMirror(
+            kw.get("ceph_cluster_dict").get("ceph-rbd2"), config
+        )
+
+        rbd1, rbd2, rbd3 = [
+            Rbd(**kw, req_cname=cluster_name)
+            for cluster_name in kw["ceph_cluster_dict"].keys()
+        ]
+
+        mirror2_image_list = rbd2.list_images(poolname)
+
+        # To run I/O on only first image
+        is_first_image = True
+        for imagename in mirror2_image_list:
+            imagespec = poolname + "/" + imagename
+            mirror2.promote(imagespec=imagespec, force=True)
+            # After site-a failure site-b promote image status will be up+stopping_replay
+            mirror2.wait_for_status(
+                imagespec=imagespec, state_pattern="up+stopping_replay"
+            )
+            if is_first_image:
+                mirror2.benchwrite(imagespec=imagespec, io=config.get("io-total"))
+                is_first_image = False
```

Comment:
We could move out this section outside the for loop
IMO, we shouldn't be checking condition on each element if we are not intended to run on only one element

Reply:
yes I/O part took out at the end.

Reply:
Not able to see the module and the the keyword is_first_image in modified code.
Resolving the comment.